<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="PtclEditor::PtclList" name="ptclList" native="true"/>
        </item>
        <item>
         <widget class="PtclEditor::EmitterSetWidget" name="emitterSetWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="PtclEditor::TextureWidget" name="textureWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PtclEditor::TextureWidget</class>
   <extends>QWidget</extends>
   <header>editor\texturewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PtclEditor::EmitterSetWidget</class>
   <extends>QWidget</extends>
   <header>editor\emittersetwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PtclEditor::PtclList</class>
   <extends>QWidget</extends>
   <header>editor\ptcllist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
